product_allowed_list/models/product_list_line.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021 Camptocamp SA
+# @author: Julien Coux <julien.coux@camptocamp.com>
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ProductListLine(models.Model):
+    _name = "product.allowed.list.line"
+    _description = "Product allowed list configuration Line"
+
+    product_allowed_list_id = fields.Many2one(
+        string="Product list configuration",
+        comodel_name="product.allowed.list",
+        required=True,
+    )
+    product_template_id = fields.Many2one(
```

Comment:
```suggestion
    product_tmpl_id = fields.Many2one(
```

Since it's a refactoring, I suggest to do this change too.
This way it's consistent with the `product.product` field name